irods/tasks/install_plugins.yml

```diff
@@ -7,31 +7,33 @@
       - irods-api-plugin-netcdf
     state: absent
```

Comment:
@tedgin - Should we remove this task? "Ensure netcdf plugins not installed"

Reply:
Yes.  It's no longer needed. Those were the old plugins for iRODS 4.1.  We no longer need to remove them, since they are no longer installed anywhere.